.gitignore

```diff
@@ -5,4 +5,4 @@ archive/*
 build/*
 downloads/*
 diff/*
-dist/*
+Python-*-*-support.*.tar.gz
```

Comment:
Please also note that Python build is now entirely in the root Makefile so the patch chunk for iOS/Makefile is no longer needed. I decided to not remove it to keep the original patch intact.